### DISCUSSION TITLE
Customizable host parameter names

### DISCUSSION THREAD

**Discussion by KennyG5000** [DISTRHO/Cardinal#535] [Ideas]
This may just be me, but I would love the option to customise the host parameter names (1-24) per patch; I love using the host automation with Open Stage Control, makes the user experience so much more hands on, but as Cardinal only sends the generic 'parameter 1, 2' etc, it can be difficult to gauge what these faders control with a fair few mapped. It would be fantastic if these could be individually named by the user and exposed to the host for textual feedback via osc. Just want to say that Cardinal has been a game changer in workflow once incorporated into my Open Stage Control template, so grateful for this project!

**Comment by dromer** [comment]
I believe this is not possible in all plugin types, which might be a show stopper. (but would happily be proven wrong)
